dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsPlan.cs

```diff
@@ -53,21 +59,36 @@ public override string ToString()
         KernelArguments arguments,
         CancellationToken cancellationToken = default)
     {
-        var logger = kernel.LoggerFactory.CreateLogger(typeof(HandlebarsPlan));
+        var logger = kernel.LoggerFactory.CreateLogger(typeof(HandlebarsPlan).ToString());
```

Comment:
@TaoChenOSU I had to add this bc it was throwing a Casting error here, something like "could not convert from object to string"

Reply:
> @TaoChenOSU I had to add this bc it was throwing a Casting error here, something like "could not convert from object to string"

That should not throw. Was this in a particular test where the ILoggerFactory might have been customized to something unexpected?